### PR TITLE
Issue 10 - SVG Treatment

### PR DESCRIPTION
**Pull request by simonhearne**
I've made this as simple as possible, if a filename includes 'svg' then ignore it.

### CODE REVIEW

**Comment by simonhearne** [comment]
This is case sensitive so we should review it. Probably best to use a regex to match something like (\.|\<|\/)svg/i

File: app/public.js [hunk 1]
```diff
@@ -292,6 +293,15 @@
     return url.hostname;
   }
 
+  function isSVG(s) {
+    let hasSVG = s.url.indexOf('svg');
```
Comment (tabanliviu) [inline app/public.js]: @simonhearne INFO - if we want to check for case insensitive file extension we should use this `return (/\.svg$/i).test(s.url)`

Comment (tabanliviu) [reply app/public.js]: DONE

**Comment by tabanliviu** [comment]
@simonhearne I'll add support for data: /svg as well

**Comment by tabanliviu** [comment]
@simonhearne I don't think we support inline svgs. Can you provide an example?

**Review by tabanliviu — APPROVED** [review]

**Comment by tabanliviu** [comment]
@simonhearne please review/test

**Review by simonhearne — COMMENTED** [review]
Yes these changes are super.

**Pull request merged**